templates/workloads/partials/cf/env-controller-action-with-alb.yml

```diff
@@ -0,0 +1,14 @@
+Metadata:
+  'aws:copilot:description': "Update your environment's shared resources"
+Type: Custom::EnvControllerFunction
+Properties:
+  ServiceToken: !GetAtt EnvControllerFunction.Arn
+  Workload: !Ref WorkloadName
+  EnvStack: !Sub '${AppName}-${EnvName}'
+  Parameters:
+    - 'ALBWorkloads'
+    {{- if eq .Network.SubnetsType "PrivateSubnets"}}
+    - 'NATWorkloads'
+    {{- end}}
+  # We need to force trigger this lambda function on all deployments, so we give it a random ID as input on all event types.
+  UpdateID: {{ randomUUID }}
```

Comment:
@iamhopaul123 we can remove UpdateID right? It was mistakenly added I believe last time 🙈

Reply:
Yeah IIRC. For safety @huanjani could you double check if it works? BTW can we merge this PR? we still haven't make any changes to the lambda to accommodate with this template change i guess.

Reply:
hmm yes, if we do `EnvControllerParameters` we prob for now should not include the "NATWorkloads" parameter to the list until we fix the Lambda code 😝